code/modules/mob/living/basic/farm_animals/gorilla/gorilla.dm

```diff
@@ -23,6 +23,7 @@
 	response_harm_continuous = "thumps"
 	response_harm_simple = "thump"
 	speed = 0.5
+	melee_attack_cooldown = 1 SECONDS
```

Comment:
```suggestion
	melee_attack_cooldown = CLICK_CD_MELEE
```
pretty sure this would work? uses the conventional click cooldown macro

Reply:
> pretty sure this would work? uses the conventional click cooldown macro


there is logic in this